mathics/builtin/specialfunctions.py

```diff
@@ -858,3 +858,34 @@ def prepare_sympy(self, leaves):
 #    nargs = 3
 #    sympy_name = ''
 #    mpmath_name = ''
+
+
+class ExpIntegralEi(_MPMathFunction):
+    '''
+    <dl>
+    <dt>'ExpIntegralEi[$z$]'
+      <dd>returns the exponential integral function $Ei(z)$.
+    </dl>
+
+    >> ExpIntegralEi[2.0]
+     = 4.95423435600189016
+    '''
+
+    sympy_name = ''
```

Comment:
http://reference.wolfram.com/language/ref/ExpIntegralEi.html has some special values for `ExpIntegralEi[0]` and `ExpIntegralEi[Infinity]`, but it looks like SymPy doesn't automatically return them:
```
>>> Ei(0)
Ei(0)
>>> Ei(oo)
Ei(oo)
```
Maybe we could add a couple of rules for these?